src/stravalib/model.py

```diff
@@ -769,23 +774,25 @@ class ActivityPhoto(BaseModel):
     endpoint to retrieve it
     """
 
-    athlete_id: Optional[int] = None
     activity_id: Optional[int] = None
```

Comment:
i alphabetizes these. it was driving me crazy trying to check against the API and both were in different orders. atleast now ours is in a specific order that we can follow.